packages/converter/converter.go

```diff
@@ -569,20 +570,31 @@ func InterfaceToStr(v interface{}) string {
 	case []byte:
 		str = string(v.([]byte))
 	default:
-		if reflect.TypeOf(v).String() == `decimal.Decimal` {
+		if reflect.TypeOf(v).String() == `map[string]interface {}` {
+			if out, err := json.Marshal(v); err != nil {
+				log.WithFields(log.Fields{"error": err, "type": consts.JSONMarshallError}).Error("marshalling map for jsonb")
+				return ``, err
+			} else {
+				str = string(out)
+			}
+		} else if reflect.TypeOf(v).String() == `decimal.Decimal` {
 			str = v.(decimal.Decimal).String()
 		}
 	}
-	return str
+	return str, nil
 }
 
 // InterfaceSliceToStr converts the slice of interfaces to the slice of strings
-func InterfaceSliceToStr(i []interface{}) []string {
-	var str []string
+func InterfaceSliceToStr(i []interface{}) (str []string, err error) {
```

Comment:
may be return value should be "strs" instead of "str"?

Reply:
All suggestions have been realized.